src/components/badges/ShareToTwitterIfNeeded.tsx

```diff
@@ -28,24 +37,48 @@ const wideBlock = classnames(
   gridColumn('lg:col-span-2', 'col-span-1')
 )
 
-export default function () {
+function ShareToTwitterIfNeededSuespended({
+  derivativeAddress,
+  network,
+}: {
+  derivativeAddress: string
```

Comment:
Let's extract this props into `interface` they're same

Reply:
fixed, ty